Checkpoints and self scan

Valioesi opened the pull request:
Added link to self scan in modal on checkpoints page .
Show berlin-2 campaign visualisations in profile and always show berlin visualisation by default on berlin website.

schaerta commented:
![image](https://user-images.githubusercontent.com/17318585/166644153-e7cd1115-7b1a-4e8e-acab-90d6c9107151.png)
Bei mir ist der Whitespace ein bisschen sehr groß. :D Aber ist jetzt kein Dealbreaker.
Aber könntest du die Überschrift beim Modal noch mal anpassen? Die Message soll ja ein Freuen und Feiern sein, dass man die Schritte abgeschlossen hat. Vielleicht sowas wie "Vielen Dank!" oder sogar "Hurra, geschafft!"

Valioesi commented:
Huch, sehr strange der whitespace! 
Headline pass ich an. War mir auch unsicher, was ich machen soll. Hatte erst "yeah". "Vielen Dank" hatte ich überlegt, aber der erste Satz geht so los. "Hurry, geschafft" finde ich gut.

Valioesi commented:
Fixed the height of the modal for small and large screens.

I also added, that for the berlin page only berlin-2 signature count is shown in the profile
